From: Clint Byrum <clint@ubuntu.com>
Description: makes mtr look in the standard location from the 
 mysql-testsuite-5.5 package.
Forwarded: not-needed
Reviewed-by: Nicholas Bamber <nicholas@periapt.co.uk>
Last-Update: 2012-10-24
Index: mysql-5.5.29/mysql-test/lib/mtr_cases.pm
===================================================================
--- mysql-5.5.29.orig/mysql-test/lib/mtr_cases.pm	2013-01-02 15:36:15.190365953 -0800
+++ mysql-5.5.29/mysql-test/lib/mtr_cases.pm	2013-01-02 15:38:13.147449465 -0800
@@ -287,7 +287,8 @@
     else
     {
       $suitedir= my_find_dir($::basedir,
-			     ["share/mysql-test/suite",
+			     ["lib/mysql-testsuite/suite",
+			      "share/mysql-test/suite",
 			      "mysql-test/suite",
 			      "internal/mysql-test/suite",
 			      "mysql-test",
